src/bootstrap/bootstrap.py

```diff
@@ -977,6 +977,7 @@ def parse_args():
     parser.add_argument('--color', choices=['always', 'never', 'auto'])
     parser.add_argument('--clean', action='store_true')
     parser.add_argument('--json-output', action='store_true')
+    parser.add_argument('--warnings', choices=['deny', 'warn', 'default'])
```

Comment:
ah hmm, I just realized bootstrap itself allows `--warnings default` even though it's not documented - seems ok to allow it here for consistency.